code/mob/living/critter/ai/mouse.dm

```diff
@@ -8,12 +8,12 @@
 	transition_tasks += holder.get_instance(/datum/aiTask/timed/wander, list(holder, src))
 	transition_tasks += holder.get_instance(/datum/aiTask/sequence/goalbased/critter/eat, list(holder, src))
 
-/datum/aiHolder/mouse_mad
+/datum/aiHolder/mouse/mad
 	New()
 		..()
-		default_task = get_instance(/datum/aiTask/prioritizer/critter/mouse_mad, list(src))
+		default_task = get_instance(/datum/aiTask/prioritizer/critter/mouse/mad, list(src))
 
-/datum/aiTask/prioritizer/critter/mouse_mad/New()
+/datum/aiTask/prioritizer/critter/mouse/mad/New()
 	..()
 	transition_tasks += holder.get_instance(/datum/aiTask/timed/wander, list(holder, src))
```

Comment:
the parent class already has this task, as well as eat. You just want to add attack here.

Reply:
Donezo